codex-ui/src/vue/components/select/Select.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <Button
+    :icon="activeItem.icon"
+    trailing-icon="BracketsVertical"
+    secondary
+    @click="!isOpen ? togglePopover($event.currentTarget, {vertically: 'below', horizontally: 'left'}) : hide()"
```

Comment:
`toggle` means that it includes hide internally. So I would suggest to move `isOpen` check inside it.